pop-cli-for-appchains/guides/running-on-paseo-locally.md

```diff
@@ -51,68 +35,43 @@ name = "charlie"
 validator = true
 ```
 
-> Interesting Fact: We have three validator nodes so that we can use Polkadot's Warp Sync to quickly sync with the network. The minimum requirement for Warp Sync are three validator nodes.
+As you can see, the sudo account (admin of the chain) is overridden with `Alice` account. This allows us to make changes
+to Paseo if needed.
 
 Run the network:
-
 ```
 pop up parachain -f network.toml --verbose
 ```
 
-> The `--verbose` flag will allow us to see extra information such as the location of the chain spec for the local Paseo network that we are running.
-
-Paseo should now be running on your machine and producing blocks. We can now move towards setting up our parachain.
+> The `--verbose` flag provides us with extra information such as the location of the Paseo chain spec file. This will become important later so keep this terminal tab as is.
 
 <figure><img src="../.gitbook/assets/Screenshot 2024-09-24 at 12.20.38 PM.png" alt=""><figcaption><p>pop up parachain -f network.toml --verbose</p></figcaption></figure>
 
-## Setting up our parachain
+Paseo should now be running on your machine and producing blocks.
 
-For the sake of this exercise, let's create a new parachain:
 
-```
-pop new parachain
-```
+### Configure Paseo
 
-```
-┌   Pop CLI : Generate a parachain
-│
-◇  Select a template provider: 
-│  Pop 
-│
-◇  Select the type of parachain:
-│  Standard 
-│
-⚙  Template License: Unlicense
-│  
-◇  Select a specific release:
-│  Polkadot v1.14 
-│
-◇  Where should your project be created?
-│  ./my-parachain
-│
-◇  What is the symbol of your parachain token?
-│  UNIT
-│
-◇  How many token decimals?
-│  12
-│
-◆  And the initial endowment for dev accounts?
-│  1u64 << 60
-└  
-```
+As of now, Paseo local doesn't provide cores to validate parachain blocks on demand. We will have to make 2 calls to Paseo
+using `Alice` as admin account.
 
-```
-cd my-parachain
-pop build
+First, configure Paseo to set coretime cores to `1`:
+```bash
+pop call parachain --pallet Configuration --function set_coretime_cores --args "1" --url ws://localhost:57731/ --suri //Alice --sudo --skip-confirm
 ```
 
-It will take some time for the parachain to build. In the meantime, we can start setting up our accounts.
+> Note: we are calling the specified rpc port `577313` which is specified in the created `network.toml` file to interact the validator `alice`.
+
+Second, assign the core to the on demand pool:
+```bash
+pop call parachain --url ws://localhost:57731 --call 0xff004a0400000a000000040100e100 --suri //Alice --skip-confirm
```

Comment:
Why the call_data here? Is not clear what we are doing if we specify the extrinsic we are calling?